src/main/java/visualizations/BarChart.java

```diff
@@ -0,0 +1,37 @@
+package visualizations;
+
+
+import tech.tablesaw.api.Table;
+import tech.tablesaw.plotly.Plot;
+import tech.tablesaw.plotly.api.HorizontalBarPlot;
+import tech.tablesaw.plotly.api.VerticalBarPlot;
+
+/**
+ * Class that plots basic Bar charts.
+ */
+public class BarChart {
+
+    /**
+     * Plots basic bar chart given
+     * chart title, data set in Table format,
+     * x-axis name, y-axis name.
+     */
+    public static void plotBar(
```

Comment:
Sooner or later, we will need more options about the plots. I suggest you wrap everything in a  class like so

```
class BarPlotOptions
{
String colName;
String chartTitle;
}
```

and then call your function as

```
static void plot(BarPlotOptions options, Table data){

...
}

```

Reply:
see the new implementation & tell me what you think, pls.